### DISCUSSION TITLE
change column header colors on hover

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1268] [Q&A] [answered]
Howdy!  I work on a few 12-step sites that use this plug-in and have noticed that the column titles changing to white on hover, with the white background, causes them to disappear.  Any suggestions for changing the on hover color?  I am thinking CSS, but I am having difficulty finding what the element is for these.

Thanks!

**Comment by anchovie91471** [comment]
CSS is definitely the right approach. Can you give me an example and I’ll take a look.

**Comment by SIA-WEBSUP** [comment]
how about:

```
#tsml table th:hover {
    background-color:  /* bg color here */;
    color: /* text color here */;
}
```

**Reply by SIA-WEBSUP** [reply, marked as answer]
Incidentally, I don't believe the plugin's is setting the hover color. So some other plugin's CSS or some other CSS would be causing this. So you'd have to be sure the CSS you're setting is not 'overridden' somehow by some other CSS.

**Comment by ACAwebbuilder** [comment]
Thank you for these suggestions. I am on a different computer and noticing that it isn't an issue here.  I have tested on other computers and not able to replicate, so I think it was just that one computer.  I apologize for wasting your time and will continue to test to see if I find anything that might be relevant to bring to your attention.  Until then, please consider the matter resolved.

**Reply by anchovie91471** [reply]
Sounds good! We definitely don't set hover colors in our CSS. Usually it's a plugin or theme. Glad you were able to resolve it! Thanks!
